updating/index.adoc

```diff
@@ -0,0 +1,90 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. Platform administrators are automatically notified when an update is available for their cluster. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understand-update-service']
+== Understanding the OpenShift Update Service
+xref:../updating/understanding-the-update-service.adoc#understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, check the following information:
+
+* xref:../updating/understanding-the-update-service.html#unmanaged-operators_understanding-the-update-service[Understanding support policy for unmanaged Operators]
+
+['updating-clusters-overview-install-and-configure-update-service']
+== Installing and configuring the OpenShift Update Service
+xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]:  For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. However, clusters in a restricted network have no way to access public APIs for update information. For more information, check the following actions:
```

Comment:
There is duplicated description here with above section.

Reply:
@jiajliu Should I change the description of `Understanding the OpenShift Update Service` since I suppose we can't change the redirecting link?

Reply:
Yes, i think so. Currently the description in `Understanding the OpenShift Update Service` is not quite clear. `For more information, check the following information:[Understanding support policy for unmanaged Operators]` makes me think this part is more related with unmanaged operators, but it does not. And `[Understanding support policy for unmanaged Operators]` part is not available, as i addressed in last comment.

Reply:
I will address this too

Reply:
Addressed

Reply:
still checking why the `page not found` error is present

Reply:
@jiajliu I realized it was giving a `page not found` error because that page is moved to [here](https://deploy-preview-42047--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster.html#update-service-overview_updating-restricted-network-cluster). The new location doesn't have the second bullet point `Understanding support policy for unmanaged Operators`. So I had to remove it from the overview. Despite correct Xref path it is giving the page not found error because the content is moved under  `updating-restricted-network-cluster.html.....`